examples/userapi/05_conditional_dimension.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>If you really wanna show this, perhaps at the very end of the notebook? otherwise this looks like a small summary of the upcoming cells.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>